tests/server_test.py

```diff
@@ -23,6 +57,29 @@ def test_myapp_basic(fc):
     pkunit.pkre("elegant.*myapp.*srw", pkcompat.from_bytes(r.data))
 
 
+def test_simulation_redirect(fc):
+    fc.sr_get("simulationRedirect", redirect=False).assert_http_redirect("simulations")
+
+
+def test_simulation_schema(fc):
+    from pykern.pkcollections import PKDict
+    from pykern.pkunit import pkexcept
+
+    r = fc.sr_post_form(
+        "simulationSchema",
+        data=PKDict(simulationType=fc.sr_sim_type),
+    )
+
+    for k in ("model", "view"):
+        assert k in r.keys()
+
+    with pkexcept("unexpected status"):
+        fc.sr_post_form(
+            "simulationSchema",
+            data=PKDict(simulationType="xyzzy"),
```

Comment:
If we ever get a code named after [xyzzy](https://en.wikipedia.org/wiki/Xyzzy_(computing)), all our tests will break. ;-)